pkg/driver_cryptocell_310/include/psa_periph_chacha20_ctx.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2024 TU Dresden
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
```

Comment:
The copyright format is outdated, please use SPDX: https://github.com/RIOT-OS/RIOT/blob/master/CODING_CONVENTIONS.md#documentation